modules/ROOT/pages/policy-mule3-add-remove-headers.adoc

```diff
@@ -6,75 +6,75 @@ endif::[]
-Custom Policies appears listing custom policies, if there are any, that are available for the API.
-+
-. Click Add Custom Policy.
-. In Name, enter a name for the policy, for example `add request header policy`.
-. In Policy Definition, click Choose File to locate the YAML file, `add-request.yaml` in this example, that you downloaded.
-. In Policy Configuration, choose the XML file, for example, `add-request.xml`, that you downloaded.
-. Click Add.
-+
-image::gw-add-custom-pol.png[]
-+
-The policy, add request header policy in this example, appears on the Custom Policies page.
-+
-image::gw-custom-pol-list.png[]
+// == Downloading Add or Remove Header Policies
+
+//Currently, the add or remove header policies are available on Anypoint Exchange. Follow the step-by-step procedures to download, set up, and apply these policies.
+
+//To download the header policies:
+
+//. Login to Anypoint Platform and go to Exchange.
+//. Search for and download the policies, or use these URLs:
+//+
+//* Add Request Headers Policy
+//+
+//`+https://www.anypoint.mulesoft.com/exchange/?search=Add%20Request%20Headers%20Policy+`
+//+
+//* Remove Request Headers Policy
+//+
+//`+https://www.anypoint.mulesoft.com/exchange/?search=Remove%20Request%20Headers%20Policy+`
+//+
+//* Add Response Headers Policy
+//+
+//`+https://www.anypoint.mulesoft.com/exchange/?search=Add%20Response%20Headers%20Policy+`
+//+
+//* Remove Response Headers Policy
+//+
+//`+https://www.anypoint.mulesoft.com/exchange/?search=Remove%20Response%20Headers%20Policy+`
+//+
+//. Unzip each policy archive.
+
+//An XML and YAML file appears in each of the folders created by unzipping the archives. For example, the add-request-header-policy folder contains the following files:
+
+//* add-request.xml
+//* add-request.yaml
+
+//You use these files to set up the policy as described later.
+
+
+//== Setting Up and Applying Header Policies
+
+//If you have configured custom policies in Anypoint Platform, the process is basically the same for add or remove header policies. The following steps show how to set //up and apply the policies in Anypoint Platform.
+
+//To set up header policies:
+
+//. Go to the main API Administration page.
+//+
+//. On the main menu drop-down, select Custom policies. (Top right corner)
+//+
+//Custom Policies appears listing custom policies, if there are any, that are available for the API.
+//+
+//. Click Add Custom Policy.
+//. In Name, enter a name for the policy, for example `add request header policy`.
+//. In Policy Definition, click Choose File to locate the YAML file, `add-request.yaml` in this example, that you downloaded.
+//. In Policy Configuration, choose the XML file, for example, `add-request.xml`, that you downloaded.
+//. Click Add.
+//+
+//image::gw-add-custom-pol.png[]
+//+
+//The policy, add request header policy in this example, appears on the Custom Policies page.
+//+
+//image::gw-custom-pol-list.png[]
 
 === Applying the Add Request/Response Header Policies
```

Comment:
change to the new policy name "Header Injection Policy"

Reply:
Done! Thanks!